app/code/Magento/Captcha/view/frontend/layout/default.xml

```diff
@@ -13,12 +13,7 @@
                     <item name="components" xsi:type="array">
                         <item name="authenticationPopup" xsi:type="array">
                             <item name="children" xsi:type="array">
-                                <item name="captcha" xsi:type="array">
-                                    <item name="component" xsi:type="string">Magento_Captcha/js/view/checkout/loginCaptcha</item>
-                                    <item name="displayArea" xsi:type="string">additional-login-form-fields</item>
-                                    <item name="formId" xsi:type="string">user_login</item>
-                                    <item name="configSource" xsi:type="string">checkout</item>
-                                </item>
+
```

Comment:
I guess we can remove this file now, and the result will be the same as the child element configured in the PHP file now, right?